server/integration/testsuite/src/test/java/org/infinispan/server/test/rollingupgrades/HotRodRollingUpgradesIT.java

```diff
@@ -33,30 +33,7 @@
  */
 @RunWith(Arquillian.class)
 @Category({RollingUpgrades.class})
```

Comment:
The servers are all available as artifacts here https://repo1.maven.org/maven2/org/infinispan/server/infinispan-server-build/
I don't expected we to test down to 5.2, but at least the migration between the current stable and the previous one. We could allow testing of older versions behind maven profiles if required.